source/installation/deployment-considerations.txt

```diff
@@ -0,0 +1,111 @@
+.. _rm-deployment-considerations:
+
+=========================
+Deployment Considerations
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In the context of Relational Migrator, deployment considerations refers 
+to where the servers, networking equipment, and databases are located. 
+This page outlines location specific details to consider when using 
+Relational Migrator.
+
+How to Run Relational Migrator
+------------------------------
+
+The following table outlines high-level recommendations for how to 
+run Relational Migrator. Also note that the 
+:ref:`Kafka deployment model <kafka-intro>` provides greater 
+resilience and scalability and can be used with any of the 
+deployment options.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Use case 
+     - Recommended environment
+
+   * - Smaller scale or debugging and testing jobs that run for 
+       less than four hours. 
+     - Laptop / Desktop / Docker
+
+   * - Larger scale production jobs that run for more than four 
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to Run Relational Migrator
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible: 
+
+- If you're using Relational Migrator for an on-premises migration, 
+  run Relational Migrator in the same data center as the source 
+  database.
+
+- If you're using a cloud hosted database, run Relational 
+  Migrator on an EC2 instance or VM in the same 
+  :abbr:`VPC (Virtual Private Cloud)` as the source database.
+
+.. tip::
+
+   Check the sleep timeout setting for your operating system. If your 
+   machine goes to sleep during a migration, the sync job fails. 
+
+Network Considerations
+----------------------
+
+When using Relational Migrator, the system firewall on the machine or 
+server must allow inbound and outbound TCP traffic to the source
+database. The source database connection must also allow TCP traffic 
+from the IP address of the machine or server running Relational
+Migrator.
+
+VPC and Cloud Networking
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running Relational Migrator on an AWS EC2 instance, check the 
+security group and firewall configurations of the EC2 instance or VM. 
+Also check that the router table allows traffic to the database. If your 
+database is hosted on a public IP address, the EC2 
+instance needs a router table entry to an internet gateway within the 
+:abbr:`VPC (Virtual Private Cloud)`. For details on specific cloud 
+provider network configurations, see these pages:
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
```

Comment:
I'm not sure we have explored this enough to recommend on the official docs. We haven't had too many requests come in for this (yet) so I'd be ore comfortable leaving PrivateLink (and any specific networking solutions) out for now if that's OK!